src/notation/view/notationnavigator.h

```diff
@@ -76,16 +94,16 @@ class NotationNavigator : public AbstractNotationPaintView
     void mousePressEvent(QMouseEvent* event) override;
     void mouseMoveEvent(QMouseEvent* event) override;
 
-    void paintCursor(QPainter* painter);
     void paintPageNumbers(QPainter* painter);
 
-    void moveCanvasToRect(const RectF& viewRect);
+    bool moveCanvasToRect(const RectF& viewRect);
 
     bool isVerticalOrientation() const;
 
     PageList pages() const;
 
     RectF m_cursorRect;
```

Comment:
Please remove this rect. We don't need it anymore

Reply:
I’m confused at what you mean; the parameter is still used in the implementation.